vault/resource_identity_entity.go

```diff
@@ -18,13 +18,15 @@ func identityEntityResource() *schema.Resource {
 		Read:   identityEntityRead,
 		Delete: identityEntityDelete,
 		Exists: identityEntityExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
 
 		Schema: map[string]*schema.Schema{
 			"name": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
 				Description: "Name of the entity.",
-				ForceNew:    true,
```

Comment:
This might be optional, but a change would still require a `ForceNew`. Also, I think this should also have `Computed: true`

Reply:
It definitely does NOT require `ForceNew`, names of entities can be changed any time and only constant is object's ID.

I am not sure what are the interactions for `Computed` fields that are passed as parameters. Can you describe them @lawliet89 ?

Reply:
`Computed is often used to represent values that are not user configurable or can not be known at time of terraform plan or apply, such as date of creation or a service specific UUID. `

Anyway, good point on the `ForceNew`. I didn't realise identity is special compared with other resources.